backend/apps/owasp/api/search/project.py

```diff
@@ -30,7 +31,20 @@ def get_projects(query, attributes=None, limit=25):
         "typoTolerance": "min",
     }
 
-    return raw_search(Project, query, params)["hits"]
+    project_cache_key = f"algolia_search_{query}"
```

Comment:
Your project cache key clashes with the [issue cache key](https://github.com/OWASP/Nest/pull/19/files#diff-f70bd846c1b9929e4c147f39efca83fa890cbb9ae173d9d767d1f468487753e2R34).